### PR TITLE
Support overview

### LINKED ISSUE #1590: /support/ to VBT

### LINKED ISSUE #1802: The site background is missing on V1 migrated pages
ubuntu.com has a [suru background image](https://assets.ubuntu.com/v1/f8a323a7-image-background-paper.png) applied to the <body> tag, this has not been reflected in the migration to Vanilla brochure theme.

We should create a class we can apply to specific pages and add it to the sections we've migrated and newly-migrated pages going forward.

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Converted the /support/ page to VBT with new content added and some new patterns/bugfixes.
 - Temp fix to the padding of inline-images (Current pattern has double the amount proposed)
 - Added back the django template tag for `<body>` classes.
 - Added a `had-background` class which gives the suru site background (This should be back-ported to other pages that have already been completed)
 - Added an x-light strip variant now that the site background isn't white
 - Added a `.u-full-width` utility class to force an element to be 100% wide.
 - Fixed some issues merging various changes to inline-images (repaired the `--compact` version on /server/maas)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support](http://0.0.0.0:8001/support)
- Check the page in all viewports
 - Check the page against the [copy](https://docs.google.com/document/d/1liY2ulrI3JiICSJquTn9cH0Hu-R6H3ufmEmYhive-QI/edit#)
 - Check that the lists on [/server/maas](http://0.0.0.0:8001/support) still look good

## Issue / Card

Fixes #1590 
Fixes #1802 

## [Demo](http://www.ubuntu.com-support-overview.demo.haus/)

### CODE REVIEW

File: templates/support/index.html [hunk 1]
```diff
@@ -1,151 +1,189 @@
...
+  {% include "templates/_nav_breadcrumb-v1.html" with section_title="Support" page_title="Overview" %}
 {% endblock second_level_nav_items %}
 
 {% block content %}
+<section class="p-strip is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Ubuntu Advantage commercial support</h1>
+      <p>Ubuntu Advantage is the commercial support package from Canonical. It includes Landscape, the Ubuntu systems management tool, and the Canonical Livepatch Service, which enables you to apply kernel fixes without restarting your Ubuntu {{lts_release_full}} systems.</p>
+      <p><a href="https://buy.ubuntu.com" class="button--primary"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>
```
Comment (pmahnke) [inline templates/support/index.html]: should be .p-button--brand

File: templates/support/index.html [hunk 2]
```diff
@@ -1,151 +1,189 @@
...
+      <li class="p-matrix__item">
+        <h3 class="p-matrix__title p-heading--four">Enterprise grade security at an affordable price</h3>
+        <p class="p-matrix__desc">Access to the latest software and our proactive support
+        services, including  the <a href="/server/livepatch" class="p-matrix__link">Canonical Livepatch Service.</a></p>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title p-heading--four">IP legal assurance</h3>
+          <p class="p-matrix__desc">The
+          <a href="https://www.ubuntu.com/legal/ubuntu-advantage/assurance">
```
Comment (pmahnke) [inline templates/support/index.html]: remove https://www.ubuntu.com

File: templates/support/index.html [hunk 3]
```diff
@@ -1,151 +1,189 @@
...
+      </li>
+      <li class="p-matrix__item">
+          <h3 class="p-matrix__title p-heading--four">Certified technology</h3>
+          <p class="p-matrix__desc">All major software vendors, hardware vendors, system
+          integrators and cloud providers have certified their
+          technologies on Ubuntu.</p>
+      </li>
+      <li class="p-matrix__item">
+          <h3 class="p-matrix__title p-heading--four">Landscape server management</h3>
+          <p class="p-matrix__desc"><a href="http://www.landscape.canonical.com" class="external">Landscape</a> is the most cost-effective tool to support and
```
Comment (pmahnke) [inline templates/support/index.html]: use p-link--external

File: templates/support/index.html [hunk 4]
```diff
@@ -1,151 +1,189 @@
...
+          <h3 class="p-matrix__title p-heading--four">Livepatch</h3>
+          <p class="p-matrix__desc">
+          For greater service availability, compliance and security, Canonical&rsquo;s <a class="p-matrix__link" href="/server/livepatch">Livepatch</a> Service means kernel patches are applied without rebooting.</p>
+      </li>
+      <li class="p-matrix__item">
+          <h3 class="p-matrix__title p-heading--four">Ubuntu Advantage for public cloud</h3>
+          <p class="p-matrix__desc">Flexible Ubuntu Virtual Guest options designed for virtualised workloads on <a class="p-matrix__link" href="/cloud/public-cloud">Ubuntu-certified public clouds</a>.</p>
+      </li>
+    </ul>
+    <p class="advantage-link">
```
Comment (pmahnke) [inline templates/support/index.html]: class?  also, need more vertical space here?

File: templates/support/index.html [hunk 5]
```diff
@@ -1,151 +1,189 @@
...
+    </div>
+    <div class="col-4 p-divider__block">
+      <h4>Talk to a member of our team</h4>
+      <p>We can recommend a support package that best suits the needs of your organisation. </p>
+      <p>Ask on <a href="/support/contact-us" class="p-link--external">Contact us</a></p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p><a href="/support/contact-us">Any questions, contact us&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/support/index.html]: perhaps a button instead?  I would try and have @yaili tell you to remove it.

File: templates/shared-v1/_systems-management-tool.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<section class="p-strip is-bordered is-deep">
+  <div class="row">
+    <div class="col-6">
+      <h2>Systems management tool: Landscape</h2>
```
Comment (pmahnke) [inline templates/shared-v1/_systems-management-tool.html]: I would put this in a col-12 row over the two col-6s

File: templates/shared-v1/_systems-management-tool.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        <li class="p-list__item is-ticked">Monitored and managed machines at scale</li>
+        <li class="p-list__item is-ticked">Controlled inventory</li>
+        <li class="p-list__item is-ticked">Integration with current systems</li>
+        <li class="p-list__item is-ticked">Maintained security and compliance</li>
+        <li class="p-list__item is-ticked">Install, upgrade or downdate packages</li>
+        <li class="p-list__item is-ticked">Role-based access</li>
+        <li class="p-list__item is-ticked">Canonical livepatch service</li>
+      </ul>
+      <p>
+        <a class="external" href="https://landscape.canonical.com/">Learn more about Landscape</a>
```
Comment (pmahnke) [inline templates/shared-v1/_systems-management-tool.html]: p-link--external

File: templates/support/index.html [hunk 8]
```diff
@@ -1,151 +1,189 @@
...
-    <div class="strip-inner-wrapper">
-        <div class="eight-col">
-            <h2>Community support</h2>
-            <p>As one of the largest open source projects, with Ubuntu you can find support from a variety of sources. Take a look - you&rsquo;re likely to find an answer to every question. If you can&rsquo;t find an answer, just ask the people in our active forums.</p>
-            <p><a href="/support/community-support">Learn more about community support&nbsp;&rsaquo;</a></p>
-        </div>
-        <div class="four-col last-col text-center priority-0">
-            <img src="{{ ASSET_SERVER_URL }}039628d5-picto-community-orange.svg" width="150" alt="Community">
-        </div>
+<section id="community-support" class="p-strip is-deep">
```
Comment (pmahnke) [inline templates/support/index.html]: p-strip--light according to https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/8.%20Support?preview=support-overview.png

File: templates/support/index.html [hunk 9]
```diff
@@ -1,151 +1,189 @@
...
 
 {% block content %}
+<section class="p-strip is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Ubuntu Advantage commercial support</h1>
+      <p>Ubuntu Advantage is the commercial support package from Canonical. It includes Landscape, the Ubuntu systems management tool, and the Canonical Livepatch Service, which enables you to apply kernel fixes without restarting your Ubuntu {{lts_release_full}} systems.</p>
+      <p><a href="https://buy.ubuntu.com" class="button--primary"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>
+    </div>
+    <div class="col-5 u-align--center u-vertically-center">
```
Comment (pmahnke) [inline templates/support/index.html]: u-hidden--small?

File: templates/support/index.html [hunk 10]
```diff
@@ -1,151 +1,189 @@
...
-            </li>
-        </ul>
-        <p class="advantage-link">
-            <a href="/support/plans-and-pricing">
-                Learn more about Ubuntu Advantage&nbsp;&rsaquo;
-            </a>
-        </p>
+<section class="p-strip--x-light is-bordered is-deep">
+  <div class="row">
+    <div class="col-6 suffix-1">
```
Comment (pmahnke) [inline templates/support/index.html]: .u-hidden--small

File: templates/shared-v1/_systems-management-tool.html (hunk 6 above)
Comment (deadlight) [inline templates/shared-v1/_systems-management-tool.html]: I tried it and it doesn't look very good.

**Comment by deadlight** [comment]
@pmahnke ready for re-review

File: templates/support/index.html [hunk 11]
```diff
@@ -1,151 +1,189 @@
...
+          <h3 class="p-matrix__title p-heading--four">Livepatch</h3>
+          <p class="p-matrix__desc">
+          For greater service availability, compliance and security, Canonical&rsquo;s <a class="p-matrix__link" href="/server/livepatch">Livepatch</a> Service means kernel patches are applied without rebooting.</p>
+      </li>
+      <li class="p-matrix__item">
+          <h3 class="p-matrix__title p-heading--four">Ubuntu Advantage for public cloud</h3>
+          <p class="p-matrix__desc">Flexible Ubuntu Virtual Guest options designed for virtualised workloads on <a class="p-matrix__link" href="/cloud/public-cloud">Ubuntu-certified public clouds</a>.</p>
+      </li>
+    </ul>
+    <p>
```
Comment (pmahnke) [inline templates/support/index.html]: try

```
<div class="row" style="margin-top: 3rem;">
  <div class="col-6">
    <p>
      <a href="/support/plans-and-pricing" class="p-button--brand">Learn more about Ubuntu Advantage</a>
    </p>
  </div>
</div>
```

File: templates/shared-v1/_systems-management-tool.html (hunk 6 above)
Comment (pmahnke) [inline templates/shared-v1/_systems-management-tool.html]: fine

**Pull request merged**